Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM mcr.microsoft.com/powershell:alpine-3.12
+
+ARG CLI_VERSION=latest
+
+LABEL name="m365pnp/cli-microsoft365:${CLI_VERSION}" \
+  description="Manage Microsoft 365 and SharePoint Framework projects on any platform" \
+  homepage="https://pnp.github.io/cli-microsoft365" \
+  maintainers="Waldek Mastykarz <waldek@mastykarz.nl>, \
+  Velin Georgiev <velin.georgiev@gmail.com>, \
+  Garry Trinder <garry.trinder@live.com>, \
+  Albert-Jan Schot <appie@digiwijs.nl>, \
+  Rabia Williams <rabiawilliams@gmail.com>, \
+  Mark Powney <mpowney@icloud.com>, \
+  Patrick Lamber <patrick@nubo.eu>" \
+  com.azure.dev.pipelines.agent.handler.node.path="/usr/bin/node"
+
+RUN apk add --no-cache \
+  curl \
+  sudo \
+  bash \
+  shadow \
+  bash-completion \
+  nodejs=12.18.4-r0 \
```

Comment:
Are we using a preview version of Node?

Reply:
This version refers to `apk` package version of node to pull, having checked the container it is running `v12.8.4`, not a release version.

![image](https://user-images.githubusercontent.com/11563347/101662778-d4a80b00-3a41-11eb-85fd-72a627c66b2a.png)

Reply:
Thanks for double checking. Odd that the versions are not aligned, but good to know we have the right thing

Reply:
Yep, I thought the same but that's how they do it 🤷🏻‍♂️😁